leaflet/index.d.ts

```diff
@@ -1292,8 +1346,10 @@ declare namespace L {
         addHandler(name: string, HandlerClass: () => Handler): this; // HandlerClass is actually a constructor function, is this the right way?
         remove(): this;
         createPane(name: string, container?: HTMLElement): HTMLElement;
-        getPane(pane: string): HTMLElement;
-        getPane(pane: HTMLElement): HTMLElement;
+        /**
+         * Name of the pane or the pane as HTML-Element
+         */
+        getPane(pane: string | HTMLElement): HTMLElement;
```

Comment:
You have a good point. I wonder if we should leave it like this or change the name of the `Content` type to something more general.